test/spectests/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+if(HUNTER_ENABLED)
+    hunter_add_package(nlohmann_json)
+endif()
+find_package(nlohmann_json REQUIRED)
+
+set(fizzy_include_dir ${PROJECT_SOURCE_DIR}/lib/fizzy)
```

Comment:
Set this in `test/CMakeList.txt`.

Reply:
Moved to test `test/CMakeList.txt` and also removed this line from `bench`